Dockerfiles/Dockerfile.bcda_prod

```diff
@@ -1,7 +1,32 @@
+# syntax = docker/dockerfile:1-experimental
+FROM golang:1.15-alpine3.12 AS builder
+
+RUN apk update upgrade
+
+WORKDIR /go/src/github.com/CMSgov/bcda-app
+
+# Download all dependencies
+# By pulling this step out, we can leverage Docker caching
+# since our tooling/dependencies do not frequently change.
+COPY go.mod go.sum tools.go ./
+RUN --mount=type=cache,target=/go/pkg/mod \
+    go mod download
+RUN --mount=type=cache,target=/go/pkg/mod \ 
+    cat tools.go | awk '/[\/\/] bcda/{flag=1; next} /[\/\/] end/{flag=0} flag' | awk -F '"' '{print $2}' | xargs -tI % go get % 
+
+COPY . .
+WORKDIR /go/src/github.com/CMSgov/bcda-app/bcda
+RUN --mount=type=cache,target=/go/pkg/mod \
+    --mount=type=cache,target=/root/.cache/go-build \
+    go install -v 
+
 # syntax = docker/dockerfile:1-experimental
 FROM golang:1.15-alpine3.12 
+ARG ENVIRONMENT
+# only add dev packages if the environment argument was set to development
+RUN [ "$ENVIRONMENT" = "development" ] && apk add git openssl entr bash
```

Comment:
Shouldn't this really be in the start script instead of the Dockerfile?  We should be using the same images locally as we would deploy to production and apk could be run once locally if we need to add some special tooling there.

Reply:
Are you suggesting we run the `apk add...` command at the start of the watch script?  If so one problem with that is the watch script is written in bash, so we need to have bask installed before we can execute it